### ISSUE TITLE
is reconstruct equals (transform + transform_inv)

### ISSUE THREAD

**Issue by jomangy** [meownoid/tensorfow-rbm#3]
Hello,
In the _README.md_ file, there is [an image that describes the API](https://github.com/meownoid/tensorfow-rbm#api). In that image the `transform_inv` arrow is facing backwards,
although for my understanding it should face forward (through the `W_T`)

If this is correct, then the following code ([taken from the last line of _bbrbm.py_ ](https://github.com/meownoid/tensorfow-rbm/blob/master/tfrbm/bbrbm.py#L39)):
`self.compute_visible_from_hidden = tf.matmul(self.y, tf.transpose(self.w)) + self.visible_bias`
should be changed to:
`self.compute_visible_from_hidden = tf.nn.sigmoid(tf.matmul(self.y, tf.transpose(self.w)) + self.visible_bias)`

thanks for a great examples and code !

**Comment by meownoid** [comment]
Yes, you are right about the wrong direction of the arrow. As for sigmoid function, it's easy to apply it afterwards if there is need for Bernoulli sampling. I believe that there are situations in which such sampling is unnecessary.

**Comment by jomangy** [comment]
Thanks, I will check for the sigmoid necessity.
